src/lib/features/feature-lifecycle/feature-lifecycle.e2e.test.ts

```diff
@@ -125,7 +125,7 @@ test('should return lifecycle stages', async () => {
         bucket: {
             toggles: {
                 my_feature_a: 'irrelevant',
-                non_existent_feature: 'irrelevent',
+                non_existent_feature: 'irrelevant',
```

Comment:
:detective: :medal_sports:

Reply:
This change is so meta I don't even know how to respond to this

Reply:
You don't say. :)